pyserini/search/_searcher.py

```diff
@@ -67,8 +67,8 @@ def from_prebuilt_index(cls, prebuilt_index_name: str):
         print(f'Attempting to initialize pre-built index {prebuilt_index_name}.')
         try:
             index_dir = download_prebuilt_index(prebuilt_index_name)
-        except ValueError:
-            print(f'Unknown pre-built index: {prebuilt_index_name}')
+        except ValueError as e:
+            print(str(e))
```

Comment:
Shouldn't we raise an exception?

Reply:
I think we're good here because we return a `None`. This is so that the caller can just check for `None` instead of needing to trap an exception.